packages/moonstone/ToggleButton/ToggleButton.module.less

```diff
@@ -10,53 +10,94 @@
 	text-align: center;
 	position: relative;
 
-	&::after {
+	&::before, &::after {
```

Comment:
Convention is to have one selector per line so let's move `&::after` to the next line.